src/Groups/pcgroup.jl

```diff
@@ -472,6 +472,211 @@ function pc_group(c::GAP_Collector)
   end
 end
 
+"""
+    exponent_vector(g::Union{PcGroupElem,SubPcGroupElem})
+
+Return the exponent vector of `g` as a list of integers, each entry corresponding to
+a group generator.
+
+# Examples
+
+```jldoctest
+julia> g = abelian_group(PcGroup, [0, 5])
+Pc group of infinite order
+
+julia> x = g[1]^-3 * g[2]^-3
+g1^-3*g2^2
+
+julia> exponent_vector(x)
+2-element Vector{Int64}:
+ -3
+  2
+```
+
+```jldoctest
+julia> gg = small_group(6, 1)
+Pc group of order 6
+
+julia> x = gg[1]^5*gg[2]^-4
+f1*f2^2
+
+julia> exponent_vector(x)
+2-element Vector{Int64}:
+ 1
+ 2
+```
+"""
+function exponent_vector(g::Union{PcGroupElem,SubPcGroupElem})
+  # check if we have a PcpGroup element
+  gObj = GapObj(g)
+  exp = if GAPWrap.IsPcpElement(gObj)
+    GAPWrap.Exponents(gObj)
+  else # finite PcGroup
+    pcgs = GAPWrap.FamilyPcgs(GapObj(parent(g)))
+    GAPWrap.ExponentsOfPcElement(pcgs, gObj)
+  end
+
+  return Vector{Int}(exp)
+end
+
+"""
+    relative_order(g::Union{PcGroupElem,SubPcGroupElem})
+
+Return the relative order of `g` as an integer, with respect to the defining generators.
```

Comment:
This should state explicitly that for generators with infinite order, we turn 0

(Optionally one could justify this: we return $n\geq 0$ such that $\langle g \rangle \cong \mathbb{Z} / n \mathbb{Z}$. Ah well)